jobs/ingestor_relp/spec

```diff
@@ -24,6 +24,7 @@ properties:
 
   logstash_ingestor.relp.port: 
     description: Port to listen for RELP messages
+    default: 2514
```

Comment:
Next time, please at least make this sort of change a separate commit. It's very unrelated to the commit message for people reviewing logs to see what changes were happening.